script.js

```diff
@@ -0,0 +1,17 @@
+var mysql = require('mysql');

```

Comment:
where is this javascript file going to be used? 
It looks like a serverside js, but your other code was going to be in python?

Reply:
it is going to be used in the website that Oded built, to display the data to the user

Reply:
Generally the client doesn't speak directly to the database.
You should be using an ORM, either with flask or django which talks to the database and presents the data to the user interface.